uint32 time_stamp

float32[4] q_sp
float32 thrust_force
